src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -15,36 +18,186 @@ namespace Microsoft.Sbom.Utils;
 /// </summary>
 public static class SPDXToSbomFormatConverterExtensions
 {
-    /// <summary>
-    /// Converts a <see cref="SPDXFile"/> object to a <see cref="SbomFile"/> object.
-    /// </summary>
-    /// <param name="spdxFile"></param>
-    /// <returns></returns>
-    public static SbomFile ToSbomFile(this File spdxFile)
+    public static SbomFile ToSbomFile(this File spdxFile, List<Element> spdx30Elements = null, List<Relationship> relationships = null)
     {
-        var checksums = spdxFile.VerifiedUsing?.Select(c => c.ToSbomChecksum());
-
-        // Not setting LicenseConcluded and LicenseInfoInFiles since the whole SBOM is required to set these values.
         return new SbomFile
         {
-            Checksum = checksums,
-            FileCopyrightText = spdxFile.CopyrightText,
+            Checksum = spdxFile.VerifiedUsing?.ToSbomChecksum(),
+            FileCopyrightText = spdxFile.CopyrightText == "NOASSERTION" ? null : spdxFile.CopyrightText,
+            Path = spdxFile.Name,
             Id = spdxFile.SpdxId,
-            Path = spdxFile.Name
+            LicenseConcluded = spdx30Elements != null && relationships != null
+            ? spdxFile.GetSingleLicense(RelationshipType.HAS_CONCLUDED_LICENSE, spdx30Elements, relationships)
```

Comment:
nit: feels like we should be checking if spdx30Elements != null and relationships != null in spdxFile.GetSingleLicense and just returning null in that case. That way, this logic could just be simplified to a call to spdxFile.GetSingleLicense. (Same below for spdxFile.GetMultipleLicenses)